parl/remote/client.py

```diff
@@ -36,11 +36,15 @@ class Client(object):
 
     """
 
-    def __init__(self, master_address):
+    def __init__(self, master_address, process_id):
         """
         Args:
             master_addr (str): ip address of the master node.
+            process_id (str): id of process that creating the `Client`.
```

Comment:
id of the process that created the `Client`.

Reply:
done